domains/transfer/Transfer.API/API.IntegrationTests/Shared/Testcontainers/WalletContainer.cs

```diff
@@ -5,7 +5,7 @@
 using Testcontainers.PostgreSql;
 using Xunit;
 
-namespace API.IntegrationTests.Testcontainers;
+namespace API.IntegrationTests.Shared.Testcontainers;
```

Comment:
Put this out of Shared. I would think Shared here refers to test of 'Shared' and not shared functionality

Reply:
Everything that is not a test of Shared functionality, get it out of the Shared folder in IntegrationTests